trajectory_msgs/JointTrajectory traj
geometry_msgs/PoseArray cartesian_states
geometry_msgs/Point[] cartesian_vel
